src/main/scala/dpla/ingestion3/OaiRecordHarvesterMain.scala

```diff
@@ -121,6 +118,11 @@ object OaiRecordHarvesterMain {
       case (Some(sets), Some(blacklist)) => {
         baseOptions + ("sets" -> (sets.split(",") filter (!blacklist.split(",").contains(_))).mkString(","))
       }
+      // No set list passed in but a blacklist was passed in
+      // TODO: Should this error out or should it try to call ListSets to generate a set list?
```

Comment:
FWIW I'm working on a refactor that will sort this all out -- it will make all four of these actions available as you laid them out through a single main method.